src/GraphQL.SystemTextJson/DocumentWriter.cs

```diff
@@ -131,7 +131,7 @@ private void ConfigureOptions(IErrorInfoProvider errorInfoProvider)
         }
 
         private static JsonSerializerOptions GetDefaultSerializerOptions(bool indent)
-            => new JsonSerializerOptions { WriteIndented = indent, PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
+            => new JsonSerializerOptions { WriteIndented = indent };
```

Comment:
Many tests failed without this change (since the naming policy was being ignored previously).

Reply:
New tests?

Reply:
Well, there's plenty of tests that already fail if this isn't 'none'.  If you want I can write an explicit test that verifies that the default naming policy is 'none'.

Reply:
Already fail? Do you mean existing tests **after** your changes?

Reply:
I mean: once the naming policy is honored by the serializer (the original bug we were trying to fix) then there are a lot of tests that fail because they rely on assuming that there is no property name conversion.

is that clear?

Reply:
ok